docs/larq/guides/bnn-architecture.md

```diff
@@ -15,19 +15,17 @@ When viewing binarization as an activation function just like ReLU, one may be i
 A typical binarized layer looks something like:
 
 ```python
-import larq as lq
-...
-x_out = lq.layers.QuantDense(
+x_out = larq.layers.QuantDense(
     units=1024,
-    input_quantizer=lq.quantizers.SteSign(clip_value=1.0),
-    kernel_quantizer=lq.quantizers.SteSign(clip_value=1.0),
-    kernel_constraint=lq.constraints.WeightClip(clip_value=1.0),
+    input_quantizer=larq.quantizers.SteSign(clip_value=1.0),
+    kernel_quantizer=larq.quantizers.SteSign(clip_value=1.0),
+    kernel_constraint=larq.constraints.WeightClip(clip_value=1.0),
```

Comment:
Are we not promoting `import larq as lq`?

Reply:
We don't do it in any of the other guides. It made more sense to me to do things consistently in the guides but I am open to reverting it back. The `bnn-architecture` guide just feels a little inconsistent allround with the rest of the laq docs though and this was some low hanging fruit to reduce that.